bin/varnishd/cache/cache_vpi.c

```diff
@@ -186,3 +186,56 @@ VPI_Call_End(VRT_CTX, unsigned n)
 	AN(vbm);
 	vbit_clr(vbm, n);
 }
+
+/*--------------------------------------------------------------------
+ * tell interface.
+ *
+ * we all agree it does not quite fit the purpose of VPI, but it fits here
+ * better than anywhere else
+ *
+ * XXX: Replace instance info with a tree indexed by name
+ */
```

Comment:
It probably belongs to `cache_vcl.c`, just like sending events happens from there, alongside `vcl_cli_tell()`.